### PR TITLE
Add public memorial search form

### PR DESCRIPTION
**Pull request by kehh**

### CODE REVIEW

**Comment by kehh** [comment]
Requires https://github.com/rwahs/providence-configuration/pull/27

**Comment by leftclickben** [comment]
Sorry, yep this looks fine.

**Comment by kehh** [comment]
but I still need a display for the menu item to appear. fooo.

File: profile/rwahs.xml [hunk 1]
```diff
@@ -9525,6 +9525,59 @@
...
+            <setting name="label" locale="en_AU">Name</setting>
+          </settings>
+        </placement>
+        <placement code="Location">
+          <bundle>ca_place_labels.name</bundle>
+          <settings>
+            <setting name="label" locale="en_AU">Location</setting>
+          </settings>
+        </placement>
+        <placement code="subject">
```
Comment (leftclickben) [inline profile/rwahs.xml]: Presumably this requires https://github.com/rwahs/providence-configuration/pull/27

**Comment by leftclickben** [comment]
That's easy enough, just copy paste the ones that are there.  That's in the config repo anyway so I assumed that would come separately (and you can just do one PR for both menu items)

**Comment by kehh** [comment]
The menu items are there, they require that both the display and the search form exist (which is good). Creating displays now.

**Comment by leftclickben** [comment]
Hmmm that logic sounds strangely familiar, almost like I wrote it myself...

**Pull request merged**